### PR TITLE
Auto Installation

### PR DESCRIPTION
**Pull request by destromas1**

### CODE REVIEW

**Comment by destromas1** [comment]
https://github.com/mapic/mapic/issues/1

File: install-to-localhost.sh [hunk 1]
```diff
@@ -3,6 +3,10 @@
 # get working dir
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+echo $DIR
+
+echo "Pulling Submodules --------------------------------->"
```
Comment (knutole) [inline install-to-localhost.sh]: it's cleaner without the --------------->

**Comment by knutole** [comment]
@destromas1 Then we have to create the first user, like I showed you yesterday. 👍

File: install-to-localhost.sh (hunk 1 above)
Comment (destromas1) [reply install-to-localhost.sh]: @knutole Will remove "------>"  . I added this as when a lot of logs are in the terminal , hard to see which is what .

**Comment by destromas1** [comment]
@knutole Yup I am on it , User creation!

**Review by knutole — APPROVED** [review]

**Pull request merged**